data/country_to_region_NamePrism.tsv

```diff
@@ -0,0 +1,118 @@
+Country	Population	Region
```

Comment:
The population column is all zero. Should we just remove it?

Reply:
The placeholder column in there for compatibility with 03.process-wiki-output.py, which I'm not realizing was there for an old analysis for the school project. I'll get rid of it.